### PR TITLE
Add Lombard translation

### PR DESCRIPTION
**Pull request by polenta-e-osei**
## TL;DR

I'm proposing to merge a translation in a god's forgotten language to make feel fellow spaghetti people (Italians and part of Swiss) less isolated, to seek attention in order to help spreading in less niche circles in this geographical area and encourage who is hesitant to sign.

In case this PR receives the blessing from the maintainers, probably someone who understands the dialects of the Lombardy and surrounding areas needs to review (resources below).

## Intro and goals

Hello, I thought to translate the letter to the language of my Italian region (and part of some others, Switzerland included).

The language has roughly 3.9 millions speakers[[1]](https://en.wikipedia.org/wiki/Lombard_language), and according to the EU[[2]](https://www.europarl.europa.eu/RegData/etudes/note/join/2013/495851/IPOL-CULT_NT(2013)495851_EN.pdf) and the UNESCO's _"Atlas of Languages in Danger"_[[3]](https://www.theguardian.com/news/datablog/2011/apr/15/language-extinct-endangered) is considered endangered. The language is composed by a lot of variants/dialects, all intelligible between each other.

The goal is not to be understood by more people since 99.99% (and 100% who is able to turn on a computer) of the speakers is also native of the national language, but to give a sense of locality and neighborhood to other fellow spaghetti eaters (Italian and Swiss) that may fear work and/or social repercussions at signing this letter. And to signal that this place is not a minefield as some like to pretend it is.

The secondary goal is that someone will see this PR (or translated letter if get merged) interesting/funny/cringe and will start to share to their peers (a.k.a. draw attention); maybe someone who was out of the loop these weeks will leave a signature and who already seen this letter and didn't pay attention/bother or was hesitant now is more encouraged.

## Important part

+ This is a throwaway account created only few days ago for this PR, so it doesn't irradiate much trustworthy energy, and there's nothing I can do about it.
+ It's a language completely neglected by the Swiss and Italian governments, so no way there's some automatic translation around to help.
+ Given the two previous points, this means that in case the maintainers are inclined to merge, they cannot directly review. So some [polenta](https://lmo.wikipedia.org/wiki/Pol%C3%A8nta) eaters other than me are needed to review.
+ I have attached down below some useful resources, for anyone that wanna review this.

## A bit of technical part

+ The `ISO 639-3` for the Lombard language is `lmo`.
+ Looking around doesn't exist an implementation that adopts or a specific definition of locale id that uses this language.
+ Probably a compliant locale id would be `lmo_IT`/`lmo_CH` or in case of two letters `lm_IT`/`lm_CH` (AFAIK the `lm` seems not allotted), I personally chose `lmo_IT`.
+ There's an Unicode combination allotted[[4]](https://emojipedia.org/flag-for-lombardy-it25/) for the flag, but unsurprisingly no font supports it, so the alternative would be (in case of merge) to use a small [image](https://commons.wikimedia.org/wiki/File:Flag_of_Lombardy.svg) or a simple text link.

## Resources

This translation reflects obviously the variant of my geographical area since that an official standard doesn't exist (and I don't know very well the other variants to have a more generic approach), so please try to be a little be flexible while you review. 😅

### Books

+ **Dictionaries (pdf)**
  + [Bergamasco-Italiano](https://books.google.com/books?id=tBeMfDJugDoC)
  + [Bresciano-Italiano](https://books.google.com/books?id=zy6RejWkAfAC)
  + [Cremasco-Italiano](https://books.google.com/books?id=6kgVAAAAYAAJ)
  + [Mantovano-Italiano](https://books.google.com/books?id=shEZAAAAYAAJ)
  + [Milanese-Italiano](https://books.google.com/books?id=yB4tAAAAYAAJ)
  + [Pavese-Italiano e Italiano-Pavese](https://books.google.it/books?id=IFYVAAAAYAAJ)
  + [Comasco-Italiano](https://books.google.it/books?id=5-ssAAAAYAAJ)

### Web

+ **Small Dictionaries**
  + [Brianzolo-Italiano](http://sprea.altervista.org/Diz.htm)
  + [Italiano-Milanese](http://www.cabaretmilanoduemila.it/milanesita/dizionari-italian-milanes/265-a-dizionario-italiano-milanese)
  + [Milanese-Italiano](https://pulcinella291.forumfree.it/?t=54181640)
+ **Standardisation attempts**
  + [Noeuva Ortografia Lombarda](https://nol.lengualombarda.eu/)
  + [Noeuva Ortografia Lombarda (from the Lombard Wikipedia)](https://lmo.wikipedia.org/wiki/Noeuva_Ortografia_Lombarda)
+ **Miscellanea**
  + [lingualombarda.it](https://www.lingualombarda.it/index.php/milanese/dizionari/vocabolario-italiano-milanese-dettagli-prodotto.html)
  + [Wikisource in Lombard](https://wikisource.org/wiki/Main_Page/Lumbaart)
  + [Lombard Wikipedia](https://lmo.wikipedia.org)

### CODE REVIEW

**Comment by shenlebantongying** [comment]
Galician translation suffering a missing flag too.

Need extra help to display an image rather than emoji.

https://github.com/rms-support-letter/rms-support-letter.github.io/pull/4703

**Comment by shenlebantongying** [comment]
Or, an ad-hoc solution: add two links below the flags since they have no main country (like English to USA, Chinese to china) that use it:

`Extra translations: Lombard, Galician`

**Comment by polenta-e-osei** [comment]
This looks a reasonable solution, and it can poke someone that likes to do virtual signaling and signed the other letter, since this gives an highlight spot to neglected and/or not so much recognized languages.
If this not alters too much the pipeline of the CI I can try to produce ad-hoc `png` for Lombard and Galician as compliant as possible to the aesthetic of the emoji flags.

**Comment by shenlebantongying** [comment]
> as compliant as possible to the aesthetic of the emoji flags.

I have no idea about the emoji font, but it actually shows differently now on Windows, Linux/Firefox and Chrome now...........................

image

**Comment by polenta-e-osei** [comment]
Oh... nevermind. :sweat_smile:

**Comment by shenlebantongying** [comment]
Let's try ad-hoc solution, other maintainers of this repo seconded it.

**Comment by shenlebantongying** [comment]
@polenta-e-osei It looks like this now, and I think it's ok (since there are unlikely someone else would try to implement a textual translation....)

However, I don't know why if I remove the locale (to prevent emoji flag), the page won't render, so I just move them to root dir.

@cmd410 @imachug  Sorry for bothering, but I have no idea about web dev and jekyll, for aesthetics reason, it might be improved a bit. ~~Whatever, it works for now :)~~

image

**Comment by cmd410** [comment]
@shenlebantongying 
I think it's reasonable to just use small png/svg images for all locales. That way it won't be dependent on flag being present in some font and will look uniformly on all platforms.

There is a [repo](https://github.com/lipis/flag-icon-css) with icons, think we can use that.

**Comment by shenlebantongying** [comment]
Maybe we can just change those svg file name as their local names, then change `` as `display: inline; margin:0;`?
